docs/utilities/validation.md

```diff
@@ -156,7 +156,9 @@ We support JSON schema version 4, 6, 7 and 201909 (from [jmespath-jackson librar
 
 `@Validation` annotation is used to validate either inbound events or functions' response.
 
-It will fail fast with `ValidationException` if an event or response doesn't conform with given JSON Schema.
+It will fail fast if an event or response doesn't conform with given JSON Schema. For most type of events a `ValidationException` will be thrown. But for 
```

Comment:
```suggestion
It will fail fast if an event or response doesn't conform with given JSON Schema. For most type of events a `ValidationException` will be thrown. 
```